docs/COMMANDS_AND_FLAGS.md

```diff
@@ -36,8 +36,8 @@ telegraf [flags]
 |`--section-filter`               |filter config sections to output, separator is `:`.  Valid values are `agent`, `global_tags`, `outputs`, `processors`, `aggregators` and `inputs`|
 |`--sample-config`                |print out full sample configuration|
 |`--once`                         |enable once mode: gather metrics once, write them, and exit|
-|`--test`                         |enable test mode: gather metrics once and print them|
-|`--test-wait`                    |wait up to this many seconds for service inputs to complete in test or once mode|
+|`--test`                         |enable test mode: gather metrics once and print them.<br>**No outputs are written!**|
```

Comment:
Instead of written, can we use "executed" or "used"

linter should complain that there should not be HTML in markdown. It can be on the same line as the bold should be enough.

Reply:
Yeah but it looks ugly and I allowed `<br>` for the markdown linter by injecting it in the deprecation PR. ;-P Do you insist on removing the newline?

Reply:
doh, sneaky you... I really don't like the inline HTML and we pretty much removed it everywhere last week when we did all our listing clean ups :)

Reply:
For e.g. tables that's sometimes the only way to get sensible formatting... Will remove the linebreak here...